content/about/community/project/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Open Source
+description: How we create Istio.
+weight: 40
+icon: construction
+---
+
+The Istio software is a result of the [Istio open source project](https://github.com/istio). You can learn about how we run our
+project or get involved by visiting the [community repository on GitHub](https://github.com/istio/community). There you can learn about
```

Comment:
I would suggest you make the different areas inside the community repo as a list
To learn about how we run our project or get involved, visit the [community repository on GitHub](https://github.com/istio/community) There, you can find information on:
* Our project's [engineering values](https://github.com/istio/community/blob/master/VALUES.md)